data_steward/validation/main.py

```diff
@@ -174,33 +192,50 @@ def run_validation(hpo_id, error_ignore_flag=False):
         errors = []
         results = []
         found_cdm_file_names = found_cdm_files
+
+        # Create all tables first to simplify downstream processes
+        # (e.g. ehr_union doesn't have to check if tables exist)
         for cdm_file_name in common.CDM_FILES:
+            cdm_table_name = cdm_file_name.split('.')[0]
+            table_id = bq_utils.get_table_id(hpo_id, cdm_table_name)
+            bq_utils.create_standard_table(cdm_table_name, table_id, drop_existing=True)
```

Comment:
I don't think we want to drop existing. If this load fails, then we've lost the data we had also.

Reply:
Will this still work? I agree we would lose data which may have been saved in previous load, but if we don't drop existing, we may run into an error when it tries to create a table which already exists. (This may be something to refactor at a later time.)

Reply:
So shouldn't we only create tables that don't exist. The table names don't change. So we should be fine.